1906-03-26.xml

```diff
@@ -0,0 +1,5276 @@
+                            Dec.</date></p>
+                    <p>For information apply to the Agents, Alexandria, Port Said and Suez, Thos.
+                        Cook &amp; Son, Ld., Leon Heller, Cairo Agent, 24, Sharia Maghraby,
+                        (Telephone 192), Cairo; F. Tedeschi, Helouan.</p>
+                    <p>Special rates for Egyptian officials, members of Army of Occupation and their
+                        families.</p>
+                    <p><measure type="indexNo">31-12-906</measure></p>
+                </div>
+                <div type="advert" xml:id="deg-ad-ell02">
+                    <head>The Ellerman Lines, Limited.</head>
+                    <head type="sub">(Including Westcott &amp; Laurance Line.)</head>
+                    <p>Regular sailings from Liverpool, Glasgow, Antwerp and London to Alexandria.
+                        Frequent sailings from Alexandria to Liverpool and London. Through freight
+                        rates to Inland towns in Great Britain also to the U.S.A</p>
+                    <table rows="4" cols="5">
+                        <row>
+                            <cell>Westcott <name>S.S. Barcelona</name></cell>
+                            <cell>due from</cell>
+                            <cell><placeName>Antwerp</placeName>, <placeName>London</placeName>
+                                &amp; <placeName>Malta</placeName></cell>
+                            <cell>about</cell>
+                            <cell><date when="1905-07-16">March 30</date></cell>
+                        </row>
+                        <row>
+                            <cell>Westcott <name>S.S. Egyptian</name></cell>
+                            <cell>due from</cell>
+                            <cell><placeName>Antwerp</placeName>, <placeName>London</placeName>
+                                &amp; <placeName>Malta</placeName></cell>
+                            <cell>about</cell>
+                            <cell><date when="1905-07-25">March 30</date></cell>
+                        </row>
+                        <row>
+                            <cell>Ellerman <name>S.S. Ararat</name></cell>
+                            <cell>due from</cell>
+                            <cell><placeName>Liverpool</placeName>, <placeName>Gibraltar</placeName>
+                                &amp; <placeName>Malta</placeName></cell>
+                            <cell>about</cell>
+                            <cell><date when="1905-07-25">March 30</date></cell>
+                        </row>
+                        <row>
+                            <cell>The Ellerman <name>S.S. Bomian</name></cell>
+                            <cell>now loading for</cell>
+                            <cell><placeName>Liverpool</placeName>
+                            </cell>
+                            <cell>will sail in a few days</cell>
+                            <cell>Taking cargo only.</c
```

Comment:
This seems to be the end of the local and general section. Add a `</div>` to close it.